### PR TITLE
More bug fixes and small enhancements

### PR DESCRIPTION
**Pull request by bmbroom**
Additional bug fixes and enhancements following after pull request #433.

### CODE REVIEW

**Review by jmelott — APPROVED** [review]

**Comment by bmbroom** [comment]
Version number will be bumped just before this pull request is merged.

File: NGCHM/WebContent/javascript/API.js [hunk 1]
```diff
@@ -37,7 +37,7 @@
     // Re-exports.
     NgChm.exportToNS ('NgChm.API', {
 	'b64toBlob': UTIL.b64toBlob,
-	'jsPDF': jspdf.jsPDF,
+	'jsPDF': (...args) => jspdf.jsPDF.apply (null, args),
```
Comment (ChrisWakefield) [inline NGCHM/WebContent/javascript/API.js:41]: Deserves a comment?

**Review by ChrisWakefield — APPROVED** [review]
Approved based on just source review with no testing performed.

File: NGCHM/WebContent/javascript/API.js (hunk 1 above)
Comment (bmbroom) [reply NGCHM/WebContent/javascript/API.js:44]: See commit c5bc524819.

**Pull request merged**